### ISSUE TITLE
a11y - Icon for toolbar button "user" - cms ui

### ISSUE THREAD

**Issue by stevepiercy** [plone/Products.CMFPlone#3866]
_Originally posted by @stevepiercy in https://github.com/plone/volto/issues/5119#issuecomment-1727737649_

I chatted with the Classic UI team. We discussed two paths, short-term and long-term.

For short-term, we agreed that two changes would be helpful:

1. Change the wording from "username" to "Settings" for the lower-left text for authenticated users.
2. Change the icon from the head and shoulders (bust, user) icon to a settings icon, either https://github.com/plone/volto/blob/master/src/icons/settings.svg for Volto or https://icons.getbootstrap.com/icons/sliders/ for Classic UI. Alternatively for Classic UI, https://icons.getbootstrap.com/icons/sliders2-vertical/.

For a long-term plan, there are some issues with registering the Site Setup that require some careful thought and planning. Ideally, `Site Settings` would have its own menu item, promoted from `username > Site Setup`, located in the lower left as a sibling to `User Settings`. This would be a good conversation to have at a sprint or PloneConf.

**Comment by Hrittik20** [comment]
Hello @stevepiercy! I would like to help with this Issue. Can you direct me to the file where I need to make the change?

**Comment by stevepiercy** [comment]
@Hrittik20 sorry, I don't know, else I would do it myself. In any case, you must read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html) and [First-time contributors](https://6.docs.plone.org/contributing/first-time.html) before you make any contribution. If you decide to work on it, then you will need to [install Plone for development](https://6.docs.plone.org/install/install-from-packages.html).

**Comment by Pawank06** [comment]
Hey, I want to work on this issue. Could you please confirm if there is anyone else working on it?

**Comment by stevepiercy** [comment]
@Pawank06 No, that is your job as the contributor. Please read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html) and [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do).
